### DISCUSSION TITLE
Zerodha iceberg order tool

### DISCUSSION THREAD

**Discussion by maikshitiz** [amit0rana/betterOptionsTrading#47] [Ideas]
Actually what happens is that Zerodha does not place market orders on Iceberg so can you make an Iceberg tool for us which allows us to place market orders on any broker

**Comment by amit0rana** [comment]
One can use betterKite to split bigger quantity into multiple market orders. That feature is already there.
If you are placing market order you really don't need iceberg simple order splitting is enough

For these features you need api key.
